### PR TITLE
Added basic auth quick start guide

### PR DESCRIPTION
**Pull request by SergeyRuzin**

### CODE REVIEW

File: BINDINGS.md [hunk 1]
```diff
@@ -360,4 +360,12 @@ Then "Search results" grid should contain no records
 [When("(.*?) element among (.*) (is| is not) expanded")]
 [Then("(.*?) element among (.*) should (be|be not) expanded")]
 Example:
-When first element among "Search results" grid rows is expanded
\ No newline at end of file
+When first element among "Search results" grid rows is expanded
+
+# Scenario hooks guides
+## Basic auth hook
+In order to make the following should be done:
```
Comment (vladimir-s-quantory) [inline BINDINGS.md:367]:
```suggestion
In order to make basic authentication work user should do the following:
```

File: BINDINGS.md [hunk 2]
```diff
@@ -360,4 +360,12 @@ Then "Search results" grid should contain no records
...
+* @BasicAuth tag should be added to scenario which requires authentication
+* AutomationConfig.json should have credentials for basic auth. Please refer to [example](Behavioral.Automation.DemoBindings/AutomationConfig.json).
+* If you want to use credentials from different sources IBasicAuthConfig interface can be used
+* If basic auth should be ignored on env then IgnoreAuth field can be used to disable basic auth
```
Comment (vladimir-s-quantory) [inline BINDINGS.md:371]:
```suggestion
* To ignore basic auth on given env the IgnoreAuth field can be used to disable basic auth
```

File: BINDINGS.md (hunk 1 above)
Comment (SergeyRuzin) [reply BINDINGS.md]: Changed

File: BINDINGS.md (hunk 2 above)
Comment (SergeyRuzin) [reply BINDINGS.md]: Changed

**Review by Brrovko — APPROVED** [review]

**Review by vladimir-s-quantory — APPROVED** [review]

**Pull request merged**